database/schema.sql

```diff
@@ -14,12 +14,13 @@ CREATE TABLE source (
 
 CREATE TABLE article (
     article_id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    article_title VARCHAR(500) NOT NULL UNIQUE,
+    article_title VARCHAR(500) NOT NULL,
     polarity_score FLOAT NOT NULL,
```

Comment:
sorry to be annoying, but I think we are now also keeping article_content (probably text as it may be large)